browser-test/src/support/admin_questions.ts

```diff
@@ -308,28 +331,46 @@ export class AdminQuestions {
     await this.expectDraftQuestionExist(questionName, questionText);
   }
 
-  async addDropdownQuestion(questionName: string,
-    options: Array<string>,
+  /** Fills out the form for a dropdown question and clicks submit.  */
+  async createDropdownQuestion({questionName,
+    options,
     description = 'dropdown description',
     questionText = 'dropdown question text',
     helpText = 'dropdown question help text',
-    enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION) {
-    await this.gotoAdminQuestionsPage();
-
-    await this.page.click('#create-question-button');
-    await this.page.click('#create-dropdown-question');
-    await waitForPageJsLoad(this.page);
+    enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION}: QuestionParams){
+      await this.gotoAdminQuestionsPage();
 
-    await this.fillInQuestionBasics(questionName, description, questionText, helpText, enumeratorName);
+      await this.page.click('#create-question-button');
+      await this.page.click('#create-dropdown-question');
 
-    for (var index in options) {
-      await this.page.click('#add-new-option');
-      var matchIndex = Number(index) + 1;
-      await this.page.fill(`:nth-match(#question-settings div.flex-row, ${matchIndex}) input`, options[index]);
-    }
+      waitForPageJsLoad(this.page);
+  
+      await this.fillInQuestionBasics(questionName, description, questionText, helpText, enumeratorName);
+  
+      for (var index in options) {
+        await this.page.click('#add-new-option');
+        var matchIndex = Number(index) + 1;
+        await this.page.fill(`:nth-match(#question-settings div.flex-row, ${matchIndex}) input`, options[index]);
+      }
+      
+      await this.clickSubmitButtonAndNavigate('Create');
+  }
 
-    await this.clickSubmitButtonAndNavigate('Create');
+  /** Changes the input field of a multi option answer. */
+  async changeMultiOptionAnswer(index: number, text: string) {
+    await this.page.fill(`:nth-match(#question-settings div.flex-row, ${index}) input`, text);

```

Comment:
suggestion (blocking): here and L353 uses the same selector logic which could be abstracted. more importantly: using presentational CSS classes in the selector, in this case `flex-row` is brittle to superficial changes in the HTML. if you need more specificity please add class(es) with semantic name(s).

Reply:
Good catch! Updated.

Reply:
Fixed.